console/network/src/canary_v0.rs

```diff
@@ -165,9 +160,17 @@ impl Network for CanaryV0 {
     /// The function name for the inclusion circuit.
     const INCLUSION_FUNCTION_NAME: &'static str = MainnetV0::INCLUSION_FUNCTION_NAME;
     /// The maximum number of certificates in a batch.
+    #[cfg(any(test, feature = "test"))] // TODO: or should we use the 'test-helpers' feature?
+    const MAX_CERTIFICATES: u16 = 25;
+    /// The maximum number of certificates in a batch.
+    #[cfg(not(any(test, feature = "test")))] // TODO: or should we use the 'test-helpers' feature?
     const MAX_CERTIFICATES: u16 = 100;
-    /// The maximum number of certificates in a batch before consensus V3 rules apply.
-    const MAX_CERTIFICATES_BEFORE_V3: u16 = 100;
+    /// The maximum number of validators in a committee.
+    #[cfg(any(test, feature = "test"))] // TODO: or should we use the 'test-helpers' feature?
+    const MAX_COMMITTEE_SIZE: [(u32, u16); 2] = [(0, 16), (4_560_000, 25)];
+    /// The maximum number of validators in a committee.
+    #[cfg(not(any(test, feature = "test")))] // TODO: or should we use the 'test-helpers' feature?
+    const MAX_COMMITTEE_SIZE: [(u32, u16); 2] = [(0, 100), (4_560_000, 100)];
```

Comment:
Is there a way to const-ify the height declarations? For example, `4_560_000` is declared 3 times instead of declared once in a unified way.

Reply:
indeed, it would be best to reference `CONSENSUS_VERSIONS` and be consistent about it

Reply:
Makes sense, updated the setup: 0b5e9f775